### PR TITLE
[ADD] Tests with odoo

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: oca/recipe/odoo/runtime/session.py [hunk 1]
```diff
@@ -228,29 +222,28 @@ def clean_environments(self, reinit=True):
...
             except StopIteration:
                 pass
             else:
-                raise RuntimeError("Called the environments context manager "
-                                   "twice and it's not finished. "
-                                   "This is really unexpected.")
-        del self._environments_gen_context
+                logger.warn(
+                    "clean_environments: we had the context manager, but "
+                    "it had not been called. This suggest low-leve "
```
Comment (hbrunn) [inline oca/recipe/odoo/runtime/session.py:234]:
```suggestion
                    "it had not been called. This suggest low-level "
```

**Review by hbrunn — CHANGES_REQUESTED** [review]

File: oca/recipe/odoo/runtime/session.py (hunk 1 above)
Comment (hbrunn) [reply oca/recipe/odoo/runtime/session.py:234]: when we touch this, let's correct the typo

Comment (StefanRijnhart) [reply oca/recipe/odoo/runtime/session.py]: Thanks, taken!

**Review by hbrunn — APPROVED** [review]
sorry, lost track of this one obviously

**Pull request merged**